recipe/build.sh

```diff
@@ -8,4 +8,4 @@ if [[ $(uname) == Linux ]]; then
   export LDFLAGS="${LDFLAGS} -Wl,--no-as-needed"
 fi
 
-UWSGI_EMBED_PLUGINS=stats_pusher_statsd ${PYTHON} -m pip install . --no-deps -vv
+UWSGI_EMBED_PLUGINS=stats_pusher_statsd ${PYTHON} -m pip install . --no-deps --no-build-isolation -vvv
```

Comment:
```suggestion
UWSGI_EMBED_PLUGINS=stats_pusher_statsd ${PYTHON} -m pip install . --no-deps --no-build-isolation -vv
```
`-vvv` is not helpful and just adds noise. Technically we just need a single `v` to get the output of the build.

Reply:
I thought I saw at least `-vv` in the majority of cases though. I will refrain from using `-vvv` then. Generally I like verbosity though.